app/utils/display-converter.js

```diff
@@ -122,6 +122,7 @@ module.exports = function (req, data, template) {
   convertedData.isTestGateway = _.get(convertedData, 'currentGatewayAccount.type') === 'test'
   convertedData.isSandbox = paymentProvider === 'sandbox'
   convertedData.isDigitalWalletSupported = paymentProvider === 'worldpay' ||
+    (paymentProvider === 'sandbox' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_SANDBOX_ACCOUNT === 'true') ||
     (paymentProvider === 'stripe' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_STRIPE_ACCOUNT === 'true') ||
     (paymentProvider === 'stripe' && convertedData.isTestGateway === true)
```

Comment:
wallet payments for stripe are now live so we can probably remove the env var that toggles this feature

```suggestion
  convertedData.isDigitalWalletSupported = paymentProvider === 'worldpay' ||
    (paymentProvider === 'sandbox' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_SANDBOX_ACCOUNT === 'true') ||
    paymentProvider === 'stripe'
```

Reply:
might need to update some tests if taking this suggestion